packages/front-end/enterprise/components/Dashboards/DashboardEditor/index.tsx

```diff
@@ -0,0 +1,486 @@
+import { ExperimentInterfaceStringDates } from "back-end/types/experiment";
+import React, { Fragment, useEffect, useRef, useState } from "react";
+import { PiCaretDownFill, PiPlus } from "react-icons/pi";
+import {
+  DashboardBlockInterfaceOrData,
+  DashboardBlockInterface,
+  DashboardBlockType,
+} from "back-end/src/enterprise/validators/dashboard-block";
+import { isDefined } from "shared/util";
+import { Flex, Heading, IconButton, Text } from "@radix-ui/themes";
+import clsx from "clsx";
+import { CREATE_BLOCK_TYPE, getBlockData } from "shared/enterprise";
+import { withErrorBoundary } from "@sentry/react";
+import Button from "@/components/Radix/Button";
+import {
+  DropdownMenu,
+  DropdownMenuItem,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+} from "@/components/Radix/DropdownMenu";
+import Tooltip from "@/components/Tooltip/Tooltip";
+import Callout from "@/components/Radix/Callout";
+import { useDefinitions } from "@/services/DefinitionsContext";
+import DashboardBlock from "./DashboardBlock";
+import DashboardBlockEditDrawer from "./DashboardBlockEditDrawer";
+import DashboardUpdateDisplay from "./DashboardUpdateDisplay";
+
+export const BLOCK_TYPE_INFO: Record<
+  DashboardBlockType,
+  { name: string; hideTitle?: boolean }
+> = {
+  markdown: {
+    name: "Markdown",
+  },
+  "metadata-description": {
+    name: "Experiment Description",
+  },
+  "metadata-hypothesis": {
+    name: "Experiment Hypothesis",
+  },
+  "variation-image": {
+    name: "Variations / Screenshots",
+  },
+  metric: {
+    name: "Metric Results",
+  },
+  dimension: {
+    name: "Dimension Results",
+  },
+  "time-series": {
+    name: "Time Series",
+  },
+  "traffic-graph": {
+    name: "Traffic Time Series",
+    hideTitle: true,
```

Comment:
ui issue: when `hideTitle` is true we are still keeping the space there but not showing either the default or the custom block title.

<img width="290" height="156" alt="Screenshot 2025-07-30 at 4 27 14 PM" src="https://github.com/user-attachments/assets/101e30f7-9571-41c6-b76a-9b2d6b450df1" />

Reply:
<img width="202" height="172" alt="Screenshot 2025-07-31 at 8 47 10 AM" src="https://github.com/user-attachments/assets/85e0d635-55de-43ee-873c-b69b7722729d" />

Oh related to Traffic Time Series (not TrafficCard tho), we do have now a box inside a box which I don't think we need.